stryker/guides/angular.md

```diff
@@ -1,58 +1,56 @@
 # Angular
 
-Stryker supports Angular projects using the Angular CLI starting from @angular/cli v6.1.0. Are you using an older version? Then there are some tips later in this document.
+Stryker 4.0 and higher supports Angular projects using the Angular CLI starting from @angular/cli v9.0.0. Are you using an older version? Then there are some tips later in this document.
 
-## @angular/cli 6.1.0 and higher
+## @angular/cli 9.0.0 and higher
 
-This setup only works with @angular/cli 6.1.0 or higher. Are you using an older version of Angular? Then we highly suggest upgrading to at least version 6.1.0 of the cli. You can use the [Angular Update Guide](https://update.angular.io/) to help you with this. If it's not possible for you to upgrade your Angular version, please check out [this repo and its commits](https://github.com/nicojs/angular-stryker-example).
+This setup only works with @angular/cli 9.0.0 or higher. Are you using an older version of Angular? Then we highly suggest upgrading to at least version 9.0.0 of the cli. You can use the [Angular Update Guide](https://update.angular.io/) to help you with this. If it's not possible for you to upgrade your Angular version, please check out [the legacy guide for Stryker 3 and Angular CLI 6.1-8.2](./legacy/stryker-3/angular.md).
 
 ## Install
 
-Install the Stryker packages using this command: `npm i -D @stryker-mutator/core @stryker-mutator/karma-runner @stryker-mutator/typescript @stryker-mutator/html-reporter`
+Either install the Stryker CLI globally using `npm install --global stryker-cli`, use `npx -p stryker-cli` in front of every Stryker command.
```

Comment:
```suggestion
Either install the Stryker CLI globally using `npm install --global stryker-cli`, use `npx` in front of every Stryker command.
```

Reply:
The dependency `@stryker-mutator/core` brings its own `stryker` command in it's `bin`, so no need for `stryker-cli` in that case.

Reply:
Thanks, that makes the instructions easier.